### PR TITLE
fix for currency icon fix

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

File: src/botPage/view/deriv/layout/Header/components/account-actions.jsx [hunk 1]
```diff
@@ -38,6 +38,14 @@ const AccountActions = () => {
     const dropdownRef = React.useRef();
     const dispatch = useDispatch();
 
+    const getBalance = () => {
+        if (currency && Number(currency) !== 0) {
+            return balance.toLocaleString(undefined, {
+                minimumFractionDigits: currencyNameMap[currency]?.fractional_digits ?? 2
```
Comment (sara-fs) [inline src/botPage/view/deriv/layout/Header/components/account-actions.jsx:44]: How can we make sure about `2` as the minimum fraction digit?

File: static/css/bot.scss [hunk 2]
```diff
@@ -141,8 +141,14 @@ body {
         word-break: break-word;
         }
     }
-
-
+#center,
+#workspace_center {
```
Comment (sara-fs) [inline static/css/bot.scss:145]: Where is it being used?

File: src/botPage/view/deriv/layout/Header/components/account-actions.jsx (hunk 1 above)
Comment (mehrzad-fs) [reply src/botPage/view/deriv/layout/Header/components/account-actions.jsx:44]: > How can we make sure about `2` as the minimum fraction digit?

Our default fraction is 2 for cryptos it's different, we hardcoding them in on our projects

File: src/botPage/view/deriv/layout/Header/components/account-actions.jsx [hunk 3]
```diff
@@ -38,6 +38,14 @@ const AccountActions = () => {
     const dropdownRef = React.useRef();
     const dispatch = useDispatch();
 
+    const getBalance = () => {
```
Comment (mehrzad-fs) [inline src/botPage/view/deriv/layout/Header/components/account-actions.jsx:41]: This change is out of this scope 
We have update the code and review this function in previous PRs

File: src/botPage/view/deriv/layout/Header/components/account-actions.jsx [hunk 4]
```diff
@@ -74,11 +82,10 @@ const AccountActions = () => {
                 <img
                     id="header__acc-icon"
                     className="header__acc-icon"
-                    src={`image/deriv/currency/ic-currency-${is_virtual ? "virtual" : currency.toLowerCase()}.svg`}
+                    src={`image/deriv/currency/ic-currency-${is_virtual ? "virtual" : currency.toLowerCase() || "unknown"}.svg`}
                 />
                 <div id="header__acc-balance" className="header__acc-balance">
-                    {balance.toLocaleString(undefined, { minimumFractionDigits: currencyNameMap[currency]?.fractional_digits ?? 2 })}
-                    <span className="symbols">&nbsp;{currency}</span>
+                    <span className="symbols">&nbsp;{getBalance()}</span>
```
Comment (mehrzad-fs) [inline src/botPage/view/deriv/layout/Header/components/account-actions.jsx:88]: same here

File: src/botPage/view/deriv/layout/Header/components/tab-content.jsx [hunk 5]
```diff
@@ -56,7 +57,13 @@ const TabContent = ({ tab, isActive, setIsAccDropdownOpen }) => {
...
             .sort((acc, acc1) => { return acc === active_account_name ? -1 : (acc1 === active_account_name ? 1 : 0) })
             .map((acc, index) => {
-              const account = accounts[acc]
+              const account = accounts[acc];
+              const has_currency = !!account.currency;
+              const display_balance = account?.balance?.toLocaleString(undefined, {
+                minimumFractionDigits:
+                  currencyNameMap[account.currency]
+                    ?.fractional_digits ?? 2,
+              }) || 0;
```
Comment (mehrzad-fs) [inline src/botPage/view/deriv/layout/Header/components/tab-content.jsx:66]: Same here

File: static/css/bot.scss (hunk 2 above)
Comment (mehrzad-fs) [inline static/css/bot.scss:145]: Same here

**Review by mehrzad-fs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**